content/_code-samples/freeze/py/freeze_token.py

```diff
@@ -0,0 +1,37 @@
+from xrpl.clients import JsonRpcClient
+from xrpl.models import IssuedCurrencyAmount, TrustSet
+from xrpl.models.transactions import TrustSetFlag
+from xrpl.transaction import (safe_sign_and_autofill_transaction,
+                              send_reliable_submission)
+from xrpl.wallet import generate_faucet_wallet
+
+client = JsonRpcClient("https://s.altnet.rippletest.net:51234") # Connect to testnet
+
+token_name = "FOO"
+
+# Amount a trustline can handle, for this transaction it is set to 0
+value = "0"
```

Comment:
If you try setting the trustline to have a value of 0, you end up getting a `tecNO_LINE_REDUNDANT` error which means you tried to delete a trustline by setting it to default settings, but there was no trustline in the first place, so it is confused. Giving any value above 0 fixes this :)
```suggestion
value = "100"
```

Reply:
@JST5000 If the trustline was set up to handle 1200 tokens and the value in the freeze token txn is set to 100 does this mean the target will be able to transact with 1,100 tokens?